bower.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "ramda-fantasy",
+  "description": "Fantasy Land compatible types for easy integration with Ramda",
+  "version": "0.2.0",
+  "authors": [{
+    "name": "Michael Hurley", 
+    "email": "mh@buzzdecafe.com",
+    "homepage": "buzzdecafe.com"
+  }, {
+    "name": "Ludwig Magnusson",
+  }],
+  "homepage": "https://www.github.com/ramda/ramda-fantasy",
+  "license": "MIT",
+  "moduleType": [
+    "node"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/ramda/ramda-fantasy.git"
+  },
+  "keywords": [
+    "functional", "abstract", "adt", "datatype", "type"
+  ],
```

Comment:
xyz would rewrite this as:

```json
  "keywords": [
    "functional",
    "abstract",
    "adt",
    "datatype",
    "type"
  ],
```

If you plan to merge #41 (or the alternative PR), you might like to reformat this list.